lib/build/bundler.js

```diff
@@ -166,7 +166,12 @@ exports.Bundler = class {
     }
 
     return doTransform()
-      .then(() => Promise.all(this.bundles.map(x => x.write(this.project.build.targets[0]))));
+      .then(() => {
+        let configTargetBundleIndex = this.bundles.findIndex(x => x.config.name == this.loaderOptions.configTarget);
```

Comment:
Just need a bit of clarification...is this just ensuring that the bundle that contains the config is processed in a certain order? I assume that is important for the hashing process. If so, can you just a comment here to explain that?